app/diagnostics.go

```diff
@@ -603,18 +612,39 @@ func (a *App) trackConfig() {
 		"enable_jenkins":                pluginActivated(cfg.PluginSettings.PluginStates, "jenkins"),
 		"enable_jira":                   pluginActivated(cfg.PluginSettings.PluginStates, "jira"),
 		"enable_nps":                    pluginActivated(cfg.PluginSettings.PluginStates, "com.mattermost.nps"),
-		"enable_nps_survey":             pluginSetting(&cfg.PluginSettings, "com.mattermost.nps", "enablesurvey", true),
+		"enable_webex":                  pluginActivated(cfg.PluginSettings.PluginStates, "com.mattermost.webex"),
```

Comment:
Why is webex on the list? It's not pre packaged. I've filled https://mattermost.atlassian.net/browse/MM-21703 to track all plugins installed via the marketplace. 1/5 that we should remove webex now and just wait for this ticket to be implemented.

Reply:
Given that the list included all (like the `aws-sns` you noted above above) I went ahead and added it. Please confirm which plugins to include for telemetry in v5.20 @aaronrothschild

Reply:
It's my understanding that we should be tracking all Mattermost plugins, whether or not they are prepackaged. I think it makes sense to add it manually here, and improve the process in the linked JIRA ticket.